doc/03-Configuration.md

```diff
@@ -63,6 +63,12 @@ do not need any manual adjustments.
 | max_rows_per_transaction       | **Optional.** Maximum number of rows Icinga DB is allowed to `SELECT`,`DELETE`,`UPDATE` or `INSERT` in a single transaction. Defaults to `8192`. |
 | wsrep_sync_wait                | **Optional.** Enforce [Galera cluster](#galera-cluster) nodes to perform strict cluster-wide causality checks. Defaults to `7`.                  |
 
+!!! info
```

Comment:
I would remove this information as it is an implementation detail. Sure, users **might** be surprised if there are 17 database connections, but I doubt anyone will ever count them.

Instead, we could reduce the size of the main pool by one. But I wouldn't attach too much importance to that.

Reply:
Since the size of the main pool is initialized in the library, we don't want to adjust it just because of Icinga DB's HA feature. Thus, I removed the highlighted info box.

Reply:
The example configuration would also have to be adjusted. But please just remove it completely. It is irrelevant.

Reply:
As we discussed offline, I have completely removed the extra info.

Reply:
I want to summarize our discussion: With this PR, we currently have only one connection that isn't tied to `max_connections`. However, there's a chance more could be added or changed, leaving it up to the user to determine and maintain the correct `max_connections` value if the goal is to truly limit Icinga DB database connections to that number. Therefore, if `max_connections` is intended to represent the actual maximum connections, it should be implemented accordingly, rather than documented that some connections are unaffected by that setting.